### DISCUSSION TITLE
Support Bootstrap 5 floating label

### DISCUSSION THREAD

**Discussion by genachka** [orchidjs/tom-select#305] [Ideas]
For consistency, when using this control with Bootstrap 5 where floating labels (https://getbootstrap.com/docs/5.1/forms/floating-labels/) are used with other Bootstrap styled controls, it would be great to able to do with this control. When an input, what I'm after is to show a placeholder value / label until a value is selected and then to have the label / placeholder move above the control in the same way that Bootstrap 5 does it. And same for when using a select.

i.e. an input using Bootstrap 5 floating label, when default is being shown on load:
![image](https://user-images.githubusercontent.com/7947294/147001018-d2e7fc1c-958e-4526-98c5-0f0838813515.png)

i.e. an input using Bootstrap 5 floating label, when a value has been set / typed:
![image](https://user-images.githubusercontent.com/7947294/147001103-85204ae3-6424-4d3f-9e32-216182fe6e0d.png)

**Comment by mvxproject** [comment]
Related to : https://github.com/orchidjs/tom-select/discussions/337
